applications/nrf5340_audio/README.rst

```diff
@@ -660,9 +660,15 @@ The application uses the following buttons on the supported development kit:
 +---------------+----------------------------------------------------------------------------------------+
 | Button        | Function                                                                               |
 +===============+========================================================================================+
-| **VOL-**      | Turns the playback volume down (and unmutes).                                          |
+| **VOL-**      | Depending on the moment it is pressed:                                                 |
+|               |                                                                                        |
+|               | * Long-pressed during startup: Changes the headset to the one to the left.             |
```

Comment:
```suggestion
|               | * Long-pressed during startup: Changes the headset to be the left channel.             |
```

Reply:
This is not clear to me. Let's clarify on Teams.

Reply:
Rephrased a bit.